File: sync/rest/services/create-service-webhook/create-service-webhook.5.x.cs
// Download the next-gen twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio.Clients;
using Twilio.Resources.Preview.Sync;

class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Auth Token at twilio.com/console
    var accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    var authToken = "your_auth_token";
    var client = new TwilioRestClient(accountSid, authToken);

    var service = ServiceResource.Create()
        .setFriendlyName("MySyncServiceInstance")
        .setWebhookUrl("https://example.com/sync")
        .Execute(client);

    Console.WriteLine(service.GetSid());
  }
}

File: sync/rest/services/create-service-webhook/create-service-webhook.5.x.rb
require 'twilio-ruby'

# Initialize the client
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
client = Twilio::REST::Client.new(account_sid, auth_token)

# Create the service
service = client.sync.v1.services.create(
  friendly_name: 'MySyncServiceInstance',
  webhook_url:   'https://example.com/sync'
)
puts service
